note/checkstyle-sample-note.md

```diff
@@ -222,7 +222,9 @@ Check that it's working by writing some poorly styled code:
 
     FAILURE: Build failed with an exception.
 
-### Step 5 - Check before launch
+### Step 5 - Check before launch (optional)
+
+(After experience, I found this setting affects the develop efficiency too much, so make the step optional.)
```

Comment:
what do you mean here? why I would not want to enable this? why I would want to enable this?